internal/alert/manager.go

```diff
@@ -33,7 +33,7 @@ type alertManager struct {
 }
 
 // NewManager ... Instantiates a new alert manager
-func NewManager(ctx context.Context, sc client.SlackClient) Manager {
+func NewSlackManager(ctx context.Context, sc client.SlackClient) Manager {
```

Comment:
Curious to know the nature/rationale behind changing this name

Reply:
Essentially, there's a new `NewLogManager`, keeping `NewManager` would indicate it can take in configuration of which type. Since that isn't the case now, specifying what the function can handle would be useful.